osmaxx/profile/models.py

```diff
@@ -0,0 +1,32 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core import signing
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+PROFILE_SALT = 'profile'
+
+
+class Profile(models.Model):
+    associated_user = models.OneToOneField(settings.AUTH_USER_MODEL, verbose_name=_('user'))
+    unverified_email = models.EmailField(verbose_name=_('unverified email'), max_length=200, null=True)
+
+    def activation_key(self):
+        key = signing.dumps(
+            obj=self._username_email_dict(),
+            salt=PROFILE_SALT)
+        return key
+
+    def validate_key(self, activation_key):
+        try:
+            return signing.loads(
+                activation_key,
+                salt=PROFILE_SALT,
+                max_age=timedelta(days=settings.REGISTRATION_VERIFICATION_TIMEOUT_DAYS)
```

Comment:
I did consider that, and decided against it. Days are quite common, and we're the only ones using this, so why make another setting that needs a description of what it does (or a very very long name to understand, ie `REGISTRATION_VERIFICATION_TIMEOUT_TIMEDELTA` doesn't cut it, because that is just much less clear than `...TIMEOUT_DAYS`). 

I'd rather go in the opposite direction: remove this as a setting altogether and have this inside the code as a constant.